tests/event/test_simulation.py

```diff
@@ -0,0 +1,422 @@
+#  Copyright 2023 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import List
+from unittest.mock import Mock
+
+from hathor.event.model.base_event import BaseEvent
+from hathor.event.model.event_data import EmptyData, ReorgData, SpentOutput, TxData, TxInput, TxMetadata, TxOutput
+from hathor.event.model.event_type import EventType
+from hathor.p2p.peer_id import PeerId
+from hathor.simulator import Simulator
+from tests.unittest import TestCase
+from tests.utils import zip_chunkify
+
+SIMULATOR_SEED = 9922163193306864793
+
+
+class TestEventSimulation(TestCase):
```

Comment:
Should we add some notes on how to generate these lists again? In case a future change impacts the tests, we can generate it again.

Reply:
Sure, done in https://github.com/HathorNetwork/hathor-core/pull/609/commits/46f66c7e961e8fc2d200626d7fa51bc3f64c3ba1

Reply:
Thanks!